.env.sample

```diff
@@ -1 +0,0 @@
-BASIQ_API_KEY=
```

Comment:
This file is a sample to create new env file and put api key into it. Please re-add @albatahmet

Reply:
I re-added this file